src/utils.js

```diff
@@ -272,3 +272,35 @@ exports.getElectron = async () => {
   spinner.succeed('Electron ready to use')
   return electronPath
 }
+
+exports.brotliSize = (path) => {
+  return new Promise((resolve, reject) => {
+    let size = 0
+    const pipe = fs.createReadStream(path).pipe(createBrotliCompress({
+      params: {
+        [constants.BROTLI_PARAM_QUALITY]: 11
+      }
+    }))
+    pipe.on('error', reject)
+    pipe.on('data', buf => {
+      size += buf.length
+    })
+    pipe.on('end', () => {
+      resolve(size)
+    })
+  })
+}
+
+exports.gzipSize = (path) => {
+  return new Promise((resolve, reject) => {
+    let size = 0
+    const pipe = fs.createReadStream(path).pipe(createGzip({ level: 9 }))
```

Comment:
Worth noting that the [default gzip_comp_level](https://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_comp_level) used by Nginx is "1", which brings questions:

- Should we use more realistic value `1` when comparing against `mazSize` ?
- Should we calculate and display values for both "9" and "1"?

Reply:
I think we should go for the default, worst case scenario sort of thing.

Reply:
Right now everything is behind a cdn so its gzip 9 or brotli.
I would like to keep the 9 to closer to bundlephobia and other tools.